packages/dev/core/src/Materials/materialPluginBase.ts

```diff
@@ -33,6 +33,12 @@ export class MaterialPluginBase {
     @serialize()
     public priority: number = 500;
 
+    /**
+     * Indicates that any #include directive in the plugin code must be replaced by the corresponding code.
+     */
+    @serialize()
+    public resolveIncludes: boolean = false;

```

Comment:
Is this for perf ?

Reply:
Yes, to avoid applying the process if not necessary, as it is based on regexp (most of the time, there won't be any #include to process)